app/src/main/java/edu/uc/brown5a7/BottomsUp/searchAdaptor/drinkAdapter.kt

```diff
@@ -18,7 +18,7 @@ class drinkAdapter(val context: Context, val drinks: ArrayList<Drink>) : Recycle
 
     inner class MyViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
 
-        var currentdrink: Drink? = null;
+        var currentdrink: Drink? = null
```

Comment:
Redundant semi-colon in Kotlin. Java practices die hard sometimes.

Reply:
Yes I agree.  :)  When I go back to Java, I forget to put the semicolon in as well.